modules/core_project_factory/scripts/preconditions/preconditions.py

```diff
@@ -431,18 +431,23 @@ def validators_for(opts, seed_project):
 
 def main(argv):
     try:
+        results = []
         opts = argparser().parse_args(argv[1:])
-        (credentials, project_id) = get_credentials(opts.credentials_path)
-        validators = validators_for(opts, project_id)
+        if opts.org_id != None:
+            (credentials, project_id) = get_credentials(opts.credentials_path)
+            validators = validators_for(opts, project_id)
 
-        results = []
-        for validator in validators:
-            results.append(validator.validate(credentials))
 
-        retcode = 0
-        for result in results:
-            if len(result["unsatisfied"]) > 0:
-                retcode = 1
+            for validator in validators:
+                results.append(validator.validate(credentials))
+
+            retcode = 0
+            for result in results:
+                if len(result["unsatisfied"]) > 0:
+                    retcode = 1
+        else:
```

Comment:
what about folder_id maybe it's ok to have folder_id but not nessesary org_id?

Reply:
We need to check org_id. There is validate function(which checks permissions) in precondition script and it fails if org_id is None. 
".terraform/modules/c158d4e24da0dc3f5e10ff9e6d98ef82/scripts/preconditions/preconditions.py", line 125, in validate
    resource = "organizations/" + self.org_id
TypeError: can only concatenate str (not "NoneType") to str